region/maridia/outer.json

```diff
@@ -1074,6 +1053,22 @@
         {
           "from": 1,
           "to": [
+            {
+              "id": 4,
+              "requires": [
+                {"or": [
+                  "canSwim",
+                  "canSuitlessMaridia"
+                ]},
+                {"canComeInCharged":{
+                  "fromNode": 1,
+                  "framesRemaining": 0,
+                  "shinesparkFrames": 150
+                }}
+              ],
+              "unlock": null,
+              "note": "Suitless shinespark direct link"
```

Comment:
Looks like the line for that link (and 4 to 1) is missing from the image.
As a side note, Mt. Everest is the biggest reason I intend to change the color of spark-only links.

Reply:
Yep missed a couple of lines but almost not sorry since it the pic becomes even messier